### ISSUE TITLE
Fix request: AvoidLombokAnnotationForNonExistentFields

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#645]
False positive. It seems that a static inner class is not recognized:
```
public class Foo {
    @Getter @Setter // <-- false positive
    public static class Config {
        private String marker;
    }
}

```
